ui/tsconfig.json

```diff
@@ -0,0 +1,104 @@
+{
+  "compilerOptions": {
```

Comment:
Please remove all of the commented out code from this file, just keep what you're actually using.

Reply:
My gut feeling is to keep these configuration options for now while the project is brand new because they are nice documentation that comes with typescript on our compiling options. I'd like to look into it more later. If we develop conventions and want some of these enabled it's very handy. Some of them about module resolution look like they may be helpful in our codebase. If you're very familiar with them though and you're very familiar with using them on typescript projects, I'll go ahead and remove it.

Reply:
Yeah I think it's ok to remove because we can just look up the docs

Reply:
I also really liked the way the file comes configured with the commented out configurations. I thought it was really cool actually that documentation was right there and I could just uncomment it to add it in. 🌟

Reply:
If it's a preferred convention to remove the unused documentation I think we can do that. I also see the convenience of keeping this for the moment while we are getting a handle on typescript. Maybe we can keep for now?

Reply:
Agreed